alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportSearchMonitorAction.kt

```diff
@@ -52,7 +53,13 @@ class TransportSearchMonitorAction @Inject constructor(
         val searchSourceBuilder = searchMonitorRequest.searchRequest.source()
         val queryBuilder = if (searchSourceBuilder.query() == null) BoolQueryBuilder()
         else QueryBuilders.boolQuery().must(searchSourceBuilder.query())
-        queryBuilder.filter(QueryBuilders.existsQuery(Monitor.MONITOR_TYPE))
+
+        if (searchMonitorRequest.searchRequest.indices().size == 1 &&
```

Comment:
this is confusing.. can we add a comment or simplify this check?

Reply:
Added a comment and simplified the check.